iteration/iteration.go

```diff
@@ -179,3 +180,25 @@ func (m *GormIterationRepository) CanStartIteration(ctx context.Context, i *Iter
 	}
 	return true, nil
 }
+
+// LoadChildren executes - select * from iterations where path <@ 'parent_path.parent_id';
+func (m *GormIterationRepository) LoadChildren(ctx context.Context, parentIterationID uuid.UUID) ([]*Iteration, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "iteration", "loadchildren"}, time.Now())
+	parentIteration, err := m.Load(ctx, parentIterationID)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
yes sure, modified error in https://github.com/almighty/almighty-core/pull/1008/commits/0419cd3eb5432f9e7201f6d00ce71d133d412a4c#diff-742f09758d56dbffa579f147b30a2be5R189